subset/bacnet/bacnetTests/.idea/.name

```diff
@@ -0,0 +1 @@
+bacnet
```

Comment:
What's this?  I"ve never seen a ".name" file before -- Not really a problem because it's in the .idea folder, but more just wondering what happens if it goes away... Can you try removing it and see if everything still works (everything being "idea" still loads the project ok).